_posts/2022-06-10-welcome-to-platform-ui-our-css-framework.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "Welcome to Platform UI - our CSS framework"
+date: 2022-06-10 10:43:13
+tags: 
+- CSS
+- Frameworks
+- RIMdev
+categories: "Platform-UI"
+twitter_text: "Platform UI a new CSS framework"
+authors: Kevin Hougasian
+image: https://images.unsplash.com/photo-1419242902214-272b3f66ee7a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2113&q=80
+image_url: https://unsplash.com/photos/ln5drpv_ImI
+image_credit: "@vincentiu"
+---
+It's been a few years, several great people, an awesome company, and a cool thing we made 😀
+
+Once upon a time, we had a mixed bag of frameworks powering our web properties. [Bootstrap](https://getbootstrap.com/), [Semantic UI](https://semantic-ui.com/), and thousands of lines written in straight CSS (and a bit of [Less](https://lesscss.org/)). Fast forward to where we have our stable CSS framework written in [Sass](https://sass-lang.com/) and a post-1.0 version that we're offering to the dev community! 
```

Comment:
I might say "Fast-forward to today, where we..."? That just flows a little better to me

Reply:
Oh, I do like that.